docs/local_setup_sans_docker.md

```diff
@@ -46,6 +77,13 @@ Now, you should be about ready to get started. This command in the top folder of
 bin/dev
 ```
 
+If you get an error related to `puma`, try uninstalling and re-installing `foreman`:
```

Comment:
📝 🌱 🤔 I did get a `puma` error, but looking at it more closely it looks like it's an error because port 5000 is already in use. Changing the `HOST` value in the env file didn't work, so looking at whether what's mentioned [here](https://medium.com/pythonistas/port-5000-already-in-use-macos-monterey-issue-d86b02edd36c) will help, since Monterey apparently uses 5000 by default

Reply:
Update - once I followed the instructions in that post (basically, Airplay uses port 5000. Disable airplay and that will free the port) everything worked!

Reply:
Ha! I ran into the same error. Had no idea what was using 5000